app/search/gleaner.py

```diff
@@ -28,48 +32,64 @@ def build_query(user_query="", page_number=1):
                 (GROUP_CONCAT(DISTINCT ?sameAs ; separator=", ") as ?sameAs)
                 (GROUP_CONCAT(DISTINCT ?keywords ; separator=", ") as ?keywords)
                 (GROUP_CONCAT(DISTINCT ?temporal_coverage ; separator=", ") as ?temporal_coverage)
+                (GROUP_CONCAT(DISTINCT ?spatial_coverage ; separator=", ") as ?spatial_coverage)
 
             {{
               VALUES ?type {{ schema:Dataset sschema:Dataset }}
               ?s a ?type .
               {{
                 ?s sschema:name ?title .
-                ?s sschema:keywords ?keywords .
-                ?s sschema:url ?url .
+                {{ ?s sschema:keywords ?keywords . }} UNION {{
+                    ?catalog ?relationship ?s .
```

Comment:
this says "if this entity ?s is a child of another node, then grab the keywords off that parent node"

Our query didn't work for CCHDO metadata as-is because it's structured like 
```
DataCatalog : {
    dataSets: [
        { set 1 }, { set 2 }
     ],
    keywords: { ice, whatever }
}
```
and we were making keywords a required field on the datasets themselves.